.github/workflows/check-links.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           jq '.devDependencies |= with_entries(
                   select(.key | test("prefix|hugo|css"))
                 )
-                | del(.dependencies, .optionalDependencies)' \
+                | del(.optionalDependencies)' \
```

Comment:
This breaks if tracing.js is always included and by that always build, so we need to decide what is more important for us (not sure yet exactly on the overall performance impact)

Reply:
This is  blocking the PR to be merged, @open-telemetry/docs-approvers please take a look